yoda.spec

```diff
@@ -1,15 +1,18 @@
-### RPM external yoda 1.5.5
+### RPM external yoda 1.6.4
 
 Source: http://cern.ch/service-spi/external/MCGenerators/distribution/%{n}/%{n}-%{realversion}-src.tgz
+#Source: http://www.hepforge.org/archive/yoda/YODA-%{realversion}.tar.gz
 
 Requires: boost python cython
+#Requires: python cython
```

Comment:
Same here, make sure that dependencies mach-up in `yoda-toolfile.spec` for SCRAM.